### PR TITLE
Features/start screen/EnterYb

### PR DESCRIPTION
**Pull request by EnterYb**

### CODE REVIEW

File: src/main/java/com/groupthree/mancala/HelloApplication.java [hunk 1]
```diff
@@ -15,9 +15,24 @@ public void start(Stage stage) throws IOException {
...
+    }
+public void startScreen(Stage stage) throws IOException {
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("start-screen-view.fxml"));
+        Scene scene = new Scene(fxmlLoader.load());
+        stage.setTitle("Group 3");
+        stage.setScene(scene);
+        stage.show();
+    }
+    public void playerDashboardScreen(Stage stage) throws IOException {
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("player-dashboard-view.fxml"));
```
Comment (mofe64) [inline src/main/java/com/groupthree/mancala/HelloApplication.java]: The methods to show the screens are not necessary for the Main class, so please remove them.
Ideally, the methods will be called from within other screens' controllers, and not the main one.

File: src/main/java/com/groupthree/mancala/controllers/PlayerDashboardController.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.groupthree.mancala.controllers;
+
+import javafx.fxml.FXML;
+
```
Comment (mofe64) [inline src/main/java/com/groupthree/mancala/controllers/PlayerDashboardController.java]: Kindly add documentation to the controller classes. We're supposed to use the Java doc format. you can refer to Java Doc slides on canvas to see a sample format.

File: src/main/java/com/groupthree/mancala/controllers/StartController.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.groupthree.mancala.controllers;
+
```
Comment (mofe64) [inline src/main/java/com/groupthree/mancala/controllers/StartController.java:2]: Kindly add documentation to the controller classes. We're supposed to use the Java doc format. you can refer to Java Doc slides on canvas to see a sample format.

**Review by mofe64 — CHANGES_REQUESTED** [review]

File: src/main/java/com/groupthree/mancala/HelloApplication.java (hunk 1 above)
Comment (EnterYb) [reply src/main/java/com/groupthree/mancala/HelloApplication.java]: @mofe64, Thanks for the review. Treated

File: src/main/java/com/groupthree/mancala/controllers/PlayerDashboardController.java (hunk 2 above)
Comment (EnterYb) [reply src/main/java/com/groupthree/mancala/controllers/PlayerDashboardController.java]: @mofe64, Thanks for the review. Treated

File: src/main/java/com/groupthree/mancala/controllers/StartController.java (hunk 3 above)
Comment (EnterYb) [reply src/main/java/com/groupthree/mancala/controllers/StartController.java:2]: @mofe64, Thanks for the review. Treated

**Review by mofe64 — APPROVED** [review]

**Pull request merged**